ecosystem/README.md

```diff
@@ -244,3 +244,57 @@ LlamaIndex is a framework for building context-augmented LLM applications.
 It chooses ollama as the LLM inference engine locally. An example can be found from the [Starter Tutorial(Local Models)](https://docs.llamaindex.ai/en/stable/getting_started/starter_example_local/).
 
 Therefore, you can integrate InternLM2 or InternLM2.5 models to LlamaIndex smoothly if you can deploying them with `ollama` as guided in the [ollama section](#ollama)
+
+
+### [LazyLLM](https://github.com/LazyAGI/LazyLLM)
+
+LazyLLM is an framework which support the easyest and lazyest way for building multi-agent LLMs applications, it offers extremely high flexibility and ease of use compared to LangChain and LLamaIndex.
+
+When you have installed `lazyllm`, you can use the following code to build chatbots based on internLM at a very low cost, without worrying about the special tokens (such as `system` and ``) of the dialogue model. Don’t worry about not having weight files; as long as you are connected to the internet, the code below will automatically download the weight files and deploy the service for you. Enjoy the convenience that LazyLLM brings to you.
+
+```python
+from lazyllm import TrainableModule, WebModule
+# Model will be download automatically if you have an internet connection
+m = TrainableModule('internlm2-chat-7b')
```

Comment:
May use 'internlm/internlm2_5-7b-chat` as an example.

Reply:
I tried this code snippet, but got failure
```
>>> from lazyllm import TrainableModule, WebModule
>>> m = TrainableModule('/mnt/140/InternLM/internlm2_5-7b-chat')
>>> WebModule(m).start().wait()
138007: 2024-07-05 11:35:39 lazyllm WARNING: (lazyllm.components.auto.auto_helper:45) Because of missing packages, the model may not run. The required packages are: ['Required: lightllm>=2.0.0', 'Required: ninja', 'Required: pyzmq', 'Required: rpyc']
Traceback (most recent call last):
  File "/home/lvhan/miniconda3/envs/test/lib/python3.10/site-packages/lazyllm/module/module.py", line 141, in _update
    top = next(stack[-1][1])
StopIteration

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/lvhan/miniconda3/envs/test/lib/python3.10/site-packages/lazyllm/module/module.py", line 169, in start
    def start(self): return self._update(mode=['server'], recursive=True)
  File "/home/lvhan/miniconda3/envs/test/lib/python3.10/site-packages/lazyllm/module/module.py", line 148, in _update
    deploy_tasks.absorb(lazyllm.call_once(top._deploy_flag, top._get_deploy_tasks)
  File "/home/lvhan/miniconda3/envs/test/lib/python3.10/site-packages/lazyllm/common/common.py", line 511, in call_once
    return func(*args, **kw)
  File "/home/lvhan/miniconda3/envs/test/lib/python3.10/site-packages/lazyllm/module/module.py", line 445, in _get_deploy_tasks
    build_deployer(self.base_model), self.url)
  File "/home/lvhan/miniconda3/envs/test/lib/python3.10/site-packages/lazyllm/module/module.py", line 434, in build_deployer
    deployer = self._deploy(base_model=base_model, stream=self._stream, **self._deploy_args)
  File "/home/lvhan/miniconda3/envs/test/lib/python3.10/site-packages/lazyllm/components/auto/autodeploy.py", line 28, in __new__
    for key, value in deploy_cls.auto_map.items():
UnboundLocalError: local variable 'deploy_cls' referenced before assignment
```

Reply:
I also tried 
```
m = TrainableModule('internlm2_5-7b-chat')
```
but got failure too
```
>>> m = TrainableModule('internlm2_5-7b-chat')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/lvhan/miniconda3/envs/test/lib/python3.10/site-packages/lazyllm/module/module.py", line 382, in __init__
    self.base_model = ModelDownloader(lazyllm.config['model_source']).download(base_model)
  File "/home/lvhan/miniconda3/envs/test/lib/python3.10/site-packages/lazyllm/components/utils/downloader/model_downloader.py", line 83, in download
    model_save_dir = self._do_download(model)
  File "/home/lvhan/miniconda3/envs/test/lib/python3.10/site-packages/lazyllm/components/utils/downloader/model_downloader.py", line 133, in _do_download
    return self._download_model_from_ms(model, self.cache_dir)
  File "/home/lvhan/miniconda3/envs/test/lib/python3.10/site-packages/lazyllm/components/utils/downloader/model_downloader.py", line 161, in _download_model_from_ms
    from modelscope.hub.snapshot_download import snapshot_download
ModuleNotFoundError: No module named 'modelscope'
```

Reply:
The first issue is a bug caused by the absence of either lightllm or vllm in the environment. We will fix it as soon as possible.
The second issue is because the library modelscope is not installed on the machine.

You can try installing an inference framework (lightllm or vllm), or install all dependencies (pytorch, vllm, lightllm) by running `pip install lazyllm[full]`.  Due to the numerous dependencies, the installation might take some time.

Reply:
We will also create a standard image and upload it to the image server in the future.

Reply:
Is modelscope a must?

Reply:
huggingface-hub and modelscope can replace each other, but the default is modelscope because it does not require additional network configuration